src/append_only_data.rs

```diff
@@ -109,20 +135,25 @@ impl UnpubPermissionSet {
     }
 }
 
+/// Set of published permissions for a user.
 #[derive(Copy, Clone, Serialize, Deserialize, PartialEq, PartialOrd, Ord, Eq, Hash, Debug)]
 pub struct PubPermissionSet {
+    /// `true` if the user can append.
     append: Option<bool>,
+    /// `true` if the user can manage permissions.
```

Comment:
Same here, should be
```suggestion
    /// `Some(true)` if the user can manage permissions.
```
And also probably worth mentioning that `Some(false)` _explicitly_ disallows a user to do that (even if `Anyone` has required permissions).